lua/telescope/builtin/files.lua

```diff
@@ -28,9 +33,10 @@ local escape_chars = function(string)
   })
 end
 
--- Special keys:
---  opts.search_dirs -- list of directory to search in
---  opts.grep_open_files -- boolean to restrict search to open files
+--- Search for a string in your current working directory and get results live as you type (respecting .gitignore)
+--- - Picker-specific options:
+---   - `grep_open_files`: boolean to restrict search to open files only, mutually exclusive with `search_dirs`
+---   - `search_dirs`: table of strings containing directories to search in, mutually exclusive with `grep_open_files`
```

Comment:
@Conni2461 please correct me if I'm wrong, the syntax for create fields with docgen is

```
---@field search_dirs table: desc
```

Also @Conni2461 does this auto point to opts?

Reply:
Ooooh....I may not have read those docgen docs clearly enough :laughing: I'll look at this later tomorrow lol